examples/index.html

```diff
@@ -65,31 +65,30 @@
 
     (function() {
       // basic Leaflet map setup
-      map = L.map("map").setView([51.505, -0.09], 13);
-      L.tileLayer(
-        "https://{s}.tiles.mapbox.com/v3/anishshah101.ipm9j6em/{z}/{x}/{y}.png",
-        {
-          maxZoom: 18,
-          attribution:
-            'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, ' +
-            '<a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, ' +
-            'Imagery © <a href="http://mapbox.com">Mapbox</a>',
-          id: "examples.map-i86knfo3"
-        }
-      ).addTo(map);
-
-      // create an image
-      img = L.distortableImageOverlay("example.png", {
-        suppressToolbar: false,
-        corners: [
-          new L.latLng(51.52, -0.1),
-          new L.latLng(51.52, -0.14),
-          new L.latLng(51.5, -0.1),
-          new L.latLng(51.5, -0.14)
-        ],
-        fullResolutionSrc: "large.jpg"
+      map = new L.map('map').setView([51.505, -0.09], 13);
+      L.tileLayer('https://{s}.tiles.mapbox.com/v3/anishshah101.ipm9j6em/{z}/{x}/{y}.png', {
+        maxZoom: 18,
+        attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, ' +
+          '<a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, ' +
+          'Imagery © <a href="http://mapbox.com">Mapbox</a>',
+        id: 'examples.map-i86knfo3'
       }).addTo(map);
 
+      // create an image
+      img = new L.DistortableImageOverlay(
+        'example.png', {
+          suppressToolbar: false,
```

Comment:
Lets leave out optional things. Instead, lets show an example separately of how to suppress toolbar altogether and make that a README section. Can be done in a separate PR. We don't want the user to think they have to do this. 
```suggestion

```